### DISCUSSION TITLE
Collected data during usage?

### DISCUSSION THREAD

**Discussion by ghost** [PrivateBin/PrivateBin#1387] [Q&A]
.

**Comment by elrido** [comment]
TL;DR: This is probably not the answer you are looking for. Your concern is a valid one, but does apply equally to other webservices, not just PrivateBin.

> as a user I am worried about what data related to the original poster is collected and stored in the host server along with the encrypted content.

Have a look at this [FAQ entry on What parts are or aren't encrypted in a paste?](https://github.com/PrivateBin/PrivateBin/wiki/FAQ#what-parts-are-or-arent-encrypted-in-a-paste) if you haven't, yet.

We try to minimize the amount of data collected. For example, the traffic-limiter doesn't store the IP, but an instance-unique hash of the IP and doesn't connect this to a particular paste.

An IP source address, which might get logged by the webserver, is required for IP-based communication (aka "the Internet"). Thats why there have always been anonymizing proxies and [Tor](https://www.torproject.org/) or the likes.

> is there any way to know the privacy policy of a specific privatebin host server?

Ultimately, even when an instance would publish such policies (which they can, in several ways), you'd still have to trust them to apply those, so I don't really see too much of a point in doing this and potentially lying to you.

Most webservers will (have to) collect IP logs and server admins or law enforcement agencies can therefore work out which IP created a paste. If this is a concern, you need to obfuscate your source IP, but that is out of scope for PrivateBin and will apply to any service you use.

A PrivateBin instance that runs as a Tor hidden service or some equivalent solution would protect against that concern (both source and destination IP would be obfuscated). On the other hand, users that use Tor, VPNs or proxies do usually stand out in traffic analysis and are by themselves interesting targets for those malicious actors.

A simpler strategy can be to pick an instance that is outside your own jurisdiction and/or their sphere of influence - hence the country codes displayed on the [directory](https://privatebin.info/directory/). You still have to trust the admin, but maybe are less concerned about some countries agencies?

Bit more involved: You could run your own instance to share with friends, on a single board computer or such, at home. Only yourself to trust.

**Reply by rugk** [reply]
Addendum: For obvious reasons, the "host it yourself and you are in control" also applies. As such, if you host it yourself, you know what logging applies and you can provide it to others you know. See [the general guide on what PrivateBin does not provide](https://github.com/PrivateBin/PrivateBin?tab=readme-ov-file#what-it-doesnt-provide).
